incubator/hnc/cmd/manager/main.go

```diff
@@ -36,6 +36,7 @@ import (
 	// +kubebuilder:scaffold:imports
 
 	api "sigs.k8s.io/multi-tenancy/incubator/hnc/api/v1alpha1"
+	api2 "sigs.k8s.io/multi-tenancy/incubator/hnc/api/v1alpha2"
```

Comment:
Maybe call this `v1a2` and rename the other one to `v1a1` to keep them parallel? (Here and elsewhere)